From 70aaf72ff2deec901dca92d58c73a4ad3a490061 Mon Sep 17 00:00:00 2001
From: Patrick Oetken <patrick@wargdrones.com>
Date: Thu, 2 Apr 2026 14:58:23 +0200
Subject: [PATCH] FIX: find jsoncpp using pkgconfig
Upstream-Status: Inappropriate [oe-specific]

---
 src/mavsdk/CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/mavsdk/CMakeLists.txt b/src/mavsdk/CMakeLists.txt
index 3d13899c..e3594830 100644
--- a/src/mavsdk/CMakeLists.txt
+++ b/src/mavsdk/CMakeLists.txt
@@ -5,7 +5,8 @@ cmake_policy(SET CMP0076 NEW)
 add_subdirectory(core)
 add_subdirectory(plugins)
 
-find_package(jsoncpp REQUIRED)
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(JSONCPP REQUIRED IMPORTED_TARGET jsoncpp)
 
 find_package(tinyxml2 REQUIRED)
 
@@ -17,7 +18,7 @@ target_link_libraries(mavsdk
     PUBLIC
     ${CMAKE_DL_LIBS}
     PRIVATE
-    JsonCpp::JsonCpp
+    PkgConfig::JSONCPP
     tinyxml2::tinyxml2
     LibEvents::LibEvents
     mav::mav
