From 88cd2a73a62f5f43480102a9794df76b3e6f5f7e Mon Sep 17 00:00:00 2001
From: Patrick Oetken <patrick@wargdrones.com>
Date: Thu, 2 Apr 2026 15:07:19 +0200
Subject: [PATCH] FIX: remove find LibLZMA from toplevel cmakelists
Upstream-Status: Inappropriate [oe-specific]

---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4a7cc625..77662ee0 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -45,7 +45,7 @@ if(NOT BUILD_WITHOUT_CURL)
     find_package(CURL REQUIRED)
 endif()
 
-    find_package(LibLZMA REQUIRED)
+find_package(LibLZMA REQUIRED)
 
 if(BUILD_TESTING AND (IOS OR ANDROID))
     message(STATUS "Building for iOS or Android: forcing BUILD_TESTING to FALSE...")
